tests/test_pango.py

```diff
@@ -137,3 +137,20 @@ def test_tabs_replace() -> None:
     pangocairocffi.show_layout(context, layout)
     surface.finish()
     assert compare_SVGObject_with_PangoText(temp_pango_text, filename)
+
+
+def check_t2s() -> None:
+    size = 1
+    temp_pango_text = Text("Helloworld", t2s={"world": ITALIC})
+    surface = cairocffi.SVGSurface(filename, WIDTH, HEIGTH)
+    context = cairocffi.Context(surface)
+    context.move_to(START_X, START_Y)
+    layout = pangocairocffi.create_layout(context)
+    layout.set_width(pangocffi.units_from_double(WIDTH))
+    fontdesc = pangocffi.FontDescription()
+    fontdesc.set_size(pangocffi.units_from_double(size * 10))
+    layout.set_font_description(fontdesc)
+    layout.set_markup('Hello<span style="italic">world</span>')  # yay, pango markup
+    pangocairocffi.show_layout(context, layout)
+    surface.finish()
+    assert compare_SVGObject_with_PangoText(temp_pango_text, filename)
```

Comment:
Comment (non-blocking): in the future, we should refactor this code into a helper function (it is mostly repeated from other tests).

Reply:
We should also rename all of the t2thing stuff

Reply:
Not for now. Maybe open a new issue @huguesdevimeux ?

Reply:
yes,of course (it was I was working on) (but won't finish before long)